org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/internal/wizard/project/ProjectImportConfiguration.java

```diff
@@ -62,7 +62,7 @@ public void setProjectDir(File projectDir) {
     }
 
 
-    public Property<Boolean> getOverwriteWorkspaceSettings() {
+    public Property<Boolean> overrideWorkspaceConfiguration() {
```

Comment:
I think this should be called `getOverrideWorkspaceConfiguration()`

Reply:
Fixed.